mmv1/third_party/terraform/services/storageinsights/resource_storage_insights_report_config_test.go

```diff
@@ -43,6 +43,82 @@ func TestAccStorageInsightsReportConfig_update(t *testing.T) {
 	})
 }
 
+func TestAccStorageInsightsReportConfig_parquet(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccStorageInsightsReportConfig_parquet(context),
```

Comment:
Can you add another test step here that uses the same resource and removes the parquet option and adds the CSV option?